src/demo/demo.test.js

```diff
@@ -0,0 +1,46 @@
+import { h } from 'preact'
+import { mount } from 'enzyme'
+import { expect } from 'chai'
+
+describe('Mount Demo App', () => {
+  let Demo = null
+
+  beforeEach(() => (Demo = require('./demo.js')))
+
+  describe('by mocking Onfido SDK', () => {
+    // Mock window.Onfido
+    global.Onfido = {
+      init: () => {},
+    }
+
+    it('mounts the Onfido Demo without crashing', () => {
+      // the component needs to be assigned to a lowercase variable to work!
+      const sdk = <Demo />
+      const sdkDemo = mount(<sdk />)
+      expect(sdkDemo).to.be.ok
+    })
+  })
+
+  describe('without mocking Onfido SDK', () => {
+    beforeEach(() => {
+      // when the Onfido SDK is imported Woopra needs to be mocked
+      const mockWoopraFn = jest.fn()
+
+      jest.mock('../Tracker/safeWoopra', () =>
+        jest.fn().mockImplementation(() => ({
+          init: () => mockWoopraFn,
+          config: () => mockWoopraFn,
+          identify: () => mockWoopraFn,
+          track: () => mockWoopraFn,
+        }))
+      )
+    })
+
+    it('mounts the Onfido Demo without crashing', () => {
+      // the component needs to be assigned to a lowercase variable to work!
+      const sdk = <Demo />
+      const sdkDemo = mount(<sdk />)
+      expect(sdkDemo).to.be.ok
+    })
```

Comment:
Duplicated test here same as what is in lines 16-21

Reply:
The SDK import is not really tested because token is `null`. Therefore those testing are just asserting that `Demo` is mounted. I need to find a way to mock the callback that set the token in the state. Until then, this test isn't testing anything meaningful, hence the duplication